README.md

```diff
@@ -6,6 +6,205 @@
 
 <h1 align="center">Price History Plugin</h1>
 
-<p align="center">Implementation of "Directive (EU) 2019/2161 of the European Parliament and of the Council",
-also known as the "Omnibus Directive".</p>
 
+<p align="center"><a href="https://sylius.com/plugins/" target="_blank"><img src="https://sylius.com/assets/badge-official-sylius-plugin.png" width="200"></a></p>
+
+⚙️ Installation 
+===============
+
+We encourage you to use the installation instructions based on Rector and Recipes as it is more convenient and faster.
+Below you can also find an instruction using the traditional installation process
+
+Prerequisites
+-------------
+
+You need to have installed both SyliusRecipes and SyliusRector. You can find how to install them here:
+- [Sylius/SyliusRecipes](https://github.com/Sylius/SyliusRecipes)
+- [Sylius/SyliusRector](https://github.com/Sylius/SyliusRector)
+
+Installation with Recipes and Rector
+-------------
```

Comment:
These lines should be adjusted to the length of the titles, here and in other places